Menu/Menu.js

```diff
@@ -18,6 +18,51 @@ let menuItems = [
       {each menu item as a list item}
     </ul>
   </div>
+*/
+
+const menuButton = document.querySelector(".menu-button")
+
+//menuItems.forEach(list => {
+  //menuButton.appendChild(menuComponent(list.array))
+//})
+
+menuButton.appendChild(menuComponent())
+  
+
+  function menuComponent (menuItems) {
+    const menu = document.createElement("div")
+    const unOrdered = document.createElement("ul")
+    //const students = document.createElement("li")
+    //const faculty = document.createElement("li")
+    //const whatNew = document.createElement("li")
+    //const tech = document.createElement("li")
+    //const music = document.createElement("li")
+    //const logOut = document.createElement("li")
+
+    menuItems.forEach(list => {
+      document.createElement("li")
```

Comment:
For each item in the list, you want to create the li element, set its textContent equal to the string, and then append it to the ul

Reply:
```
const li = document.createElement('li');
li.textContent = list;
ul.appendChild(li)
``